Wright_Will/Assignments/helloworld.js

```diff
@@ -0,0 +1 @@
+console.log('hello world')
```

Comment:
This was by far the most advanced piece of code I have ever seen.  Excellent work. :-p